app/assets/javascripts/sumofus/backbone/form_methods.js

```diff
@@ -64,6 +65,32 @@ const FormMethods = {
     this.$(`.${plugin_type}-bar__welcome-name`).text(member.welcome_name);
     this.$(`.${plugin_type}-bar__welcome-text`).removeClass('hidden-irrelevant');
   },
+
+  insertActionKitId(form_type, urlParams) {
+    let $form;
+    if(form_type == 'petition') {
+      $form = $('.petition-bar__main').find('form')[0];
+    } else if(form_type == 'fundraiser') {
+      $form = $('.fundraiser-bar__step-panel').find('form')[0]
+    }
+
+    if(urlParams && 'akid' in urlParams) {
+      if($form) {
+        $('<input>').attr({
+          type: 'hidden',
+          name: 'actionkit_user_id',
```

Comment:
I think tying into the mixup, this is where the `akid` from the url is being renamed an `actionkit_user_id`